### DISCUSSION TITLE
Cannot get the Script to run

### DISCUSSION THREAD

**Discussion by drsgable** [FuLoRi/Analyze-LastPassVaultGUI#18] [Q&A] [answered]
I am truly uneducated in this level of code, so please bear with me.
I have downloaded the Analyze-LastPAssVault.ps1 file into my Downloads folder.
I start PowerScript and a window opens with a command prompt C:\Users\Dan>
The instructions I read say to enter .\Analyze-LastPassVault.ps1 and press Enter to start the script

I get back an error message saying in essence that what I just entered is not valid
By trial and error, I discovered that entering "C:\Downloads\Analyze-LastPassVault.ps1" does not produce an error, but returns what I just entered and then displays another C:\Users\Dan command prompt.

What am I doing wrong?

**Comment by FuLoRi** [comment, marked as answer]
Hi @drsgable,

If you open your Downloads folder in Windows Explorer and right-click the _Analyze-LastPassVault.ps1_ file, do you see a menu option called _Run with PowerShell_? If so, try clicking that menu option. It's an easier way to run the script.

Good luck!

**Comment by drsgable** [comment]
Rob,

Thank you, that worked!

Dan…

**Reply by FuLoRi** [reply]
Dan,

Thanks for letting me know. :)

-Rob
